scripts/validate/govcms-validate-modules

```diff
@@ -11,10 +11,19 @@ set -euo pipefail
 
 GOVCMS_CONFIG_FOLDER_PATH=${GOVCMS_CONFIG_FOLDER_PATH:-"config"}
 GOVCMS_CORE_EXTENTION_FILE_NAME=${GOVCMS_CORE_EXTENTION_FILE_NAME:-"core.extension.yml"}
-GOVCMS_DISALLOWED_MODULES=${GOVCMS_DISALLOWED_MODULES:-"update,dblog"}
 GOVCMS_PREPARE_XML_SCRIPT=${GOVCMS_PREPARE_XML_SCRIPT:-govcms-prepare-xml}
 GOVCMS_OUTFILE=${GOVCMS_OUTFILE:-govcms-validate-modules}
 
+GOVCMS_REQUIRED_MODULES=(
```

Comment:
Arrays are nicer for ongoing management of this - it will keep PRs lean when we need to update these lists.